src/app/profiles/profile.module.ts

```diff
@@ -21,6 +23,7 @@ import {
   imports: [
     CommonModule,
     SharedModule,
+    MaterialUiModule,
```

Comment:
Consider adding this to the SharedModule and avoid adding it here.

Reply:
Yes, it should be moved into the SharedModule.